tcms/telemetry/templates/telemetry/testing/execution-trends.html

```diff
@@ -9,6 +9,7 @@
     <div class="container-fluid container-cards-pf">
         {% include "telemetry/include/filters.html" %}
 
+        <div id="metadata" data-total-key="{% trans 'TOTAL' %}"></div>
```

Comment:
Doesn't really matter but this can be merged with the main div in this page b/c it is only data attributes.